mls/nostr-mls/src/groups.rs

```diff
@@ -57,6 +58,25 @@ pub struct NostrGroupConfigData {
     pub image_key: Option<Vec<u8>>,
     /// Relays used by the group
     pub relays: Vec<RelayUrl>,
+    /// Group admins
+    pub admins: Vec<PublicKey>,
+}
+
+/// Configuration for updating group data with optional fields
+#[derive(Debug, Clone)]
```

Comment:
Here you can derive a `Default` and in the `NostrGroupDataUpdatenew` constructor use `Self::default()`